### PR TITLE
Update Junit to match date

### PR DESCRIPTION
**Pull request by Littleme1032**
Update Junit to match date function #80

### CODE REVIEW

**Comment by Littleme1032** [comment]
@limkongkiat @ctzeyong , im not sure if u edited the other testcase for viewing list. if u havent done it then i can add it back for you

File: src/test/java/seedu/duke/command/AddExpenseCommandTest.java [hunk 1]
```diff
@@ -13,6 +13,7 @@
```
Comment (limkongkiat) [inline src/test/java/seedu/duke/command/AddExpenseCommandTest.java:13]: Double newline here seems unnecessary, perhaps a single newline will do?

**Review by limkongkiat — APPROVED** [review]
LGTM overall, can add Javadoc if got time.

File: src/test/java/seedu/duke/command/AddIncomeCommandTest.java [hunk 2]
```diff
@@ -46,15 +49,16 @@ void tearDown() {
      * Verifies that the income is added to the financial list and that the correct output is printed.
      */
     @Test
-    void execute_addExpense_expectAddedToFinancialList() {
-        addIncomeCommand = new AddIncomeCommand(500.0, "allowance");
+    void execute_addExpense_expectAddedToFinancialList() throws FinanceBuddyException {
```
Comment (ctzeyong) [inline src/test/java/seedu/duke/command/AddIncomeCommandTest.java:52]: Should be addIncome not addExpense (might have been my fault though)

File: src/test/java/seedu/duke/command/AddIncomeCommandTest.java [hunk 3]
```diff
@@ -67,26 +71,29 @@ void execute_addExpense_expectAddedToFinancialList() {
      * Verifies that all incomes are added correctly and that the output is printed for each.
      */
     @Test
-    void execute_addMultipleExpenses_expectAllAddedToFinancialList() {
-        addIncomeCommand = new AddIncomeCommand(400, "Cost of Living payment");
+    void execute_addMultipleExpenses_expectAllAddedToFinancialList() throws FinanceBuddyException {
```
Comment (ctzeyong) [inline src/test/java/seedu/duke/command/AddIncomeCommandTest.java:74]: Same here, should be addMultipleIncomes

File: src/test/java/seedu/duke/command/AddIncomeCommandTest.java [hunk 4]
```diff
@@ -67,26 +71,29 @@ void execute_addExpense_expectAddedToFinancialList() {
...
+        String specificDate = "21/12/24";
+        String currentDate = LocalDate.now().format(DateTimeFormatter.ofPattern("dd/MM/yy"));
+        addIncomeCommand = new AddIncomeCommand(400, "Cost of Living payment", specificDate);
         addIncomeCommand.execute(financialList);
 
-        addIncomeCommand = new AddIncomeCommand(10.50, "friend return money");
+        addIncomeCommand = new AddIncomeCommand(10.50, "friend return money",null);
```
Comment (ctzeyong) [inline src/test/java/seedu/duke/command/AddIncomeCommandTest.java:80]: May have missed out a space between the comma and null?

**Review by ctzeyong — COMMENTED** [review]
Overall LGTM

**Pull request merged**